parsers/IN_UP.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+import re
+from ast import literal_eval
+from operator import itemgetter
+import json
+from arrow import get
```

Comment:
We can either do from arrow import get, now or just import arrow. No need for both.